react-toggle-switch-component/src/toggle-switch.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+class ToggleSwitch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { switchStatus: false };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState({ switchStatus: !this.state.switchStatus });
+  }
+
+  render() {
+    const switchStatus = this.state.switchStatus;
+    if (switchStatus === false) {
+      return (
+        <div onClick={this.handleClick}>
+          <div className="switch-on">
+            <div className="toggle-on"></div>
+          </div>
+          <p>ON</p>
+        </div>
+      );
+    } else if (switchStatus === true) {
```

Comment:
You never need to check the opposite of what was already checked, if it isn't `false` then it has to be `true`, so just an `else` here would be better.